config/locales/en/helpers.yml

```diff
@@ -33,11 +33,11 @@ en:
         add_another: Do you want to add another defendant?
       steps_claim_details_form:
         claim_details: Claim Details
-        prosecution_evidence: Prosecution Evidence
-        defence_statement: Defence statement
+        prosecution_evidence: Number of pages of prosecution evidence
```

Comment:
so these are for legend elements assocaited with the input (which I don;t think exist) so we can remove these entries